ci/check-if-docs-pr.sh

```diff
@@ -24,14 +24,31 @@ fi
 
 # List the files that are touched in the PR, filter for docs changes and remove
 # whitespace. If output is empty then only docs are changed.
-# Note: known limitation, see
-# opstrace-prelaunch/issues/1708#issuecomment-717404504
-
-# For debugging, how what this command really returns
-# git --no-pager diff HEAD main
-git --no-pager diff --name-only HEAD main
-
-DOCS_ONLY_CHANGES=$(git --no-pager diff --name-only HEAD main | egrep -v "^docs/|.gitattributes|^.github|^README.md|^.markdownlint.json" | tr -d '[:space:]')
+#
+# https://docs.github.com/en/free-pro-team@latest/rest/reference/pulls#list-pull-requests-files
+FILES_EDITED_IN_PR=$(
+    curl \
+    --silent \
```

Comment:
Let's maybe add args here for retrying the HTTP request upon transient errors :).